src/cr/cube/cube.py

```diff
@@ -188,19 +188,17 @@ class Cube(object):
     """Provides access to individual slices on a cube-result.
 
     It also provides some attributes of the overall cube-result.
+
+    `cube_idx` must be `None` (or omitted) for a single-cube CubeSet. This indicates the
+    CubeSet contains only a single cube and influences behaviors like CA-as-0th.
     """
 
     def __init__(
-        self,
-        response,
-        transforms=None,
-        first_cube_of_tab=False,
-        population=None,
-        mask_size=0,
+        self, response, cube_idx=None, transforms=None, population=None, mask_size=0
     ):
         self._cube_response_arg = response
         self._transforms_dict = {} if transforms is None else transforms
-        self._first_cube_of_tab = first_cube_of_tab
+        self._cube_idx_arg = cube_idx
```

Comment:
why `_cube_idx_arg` and not `cube_idx`?

Reply:
`cube_idx_arg` (the `cube_idx` argument) can be `None` or `int`. Cube index is always an int. So the `_arg` suffix cues the reader to look for some reason why the argument doesn't directly become the served value.